packages/adapter-components/src/deployment/deployment.ts

```diff
@@ -103,6 +106,7 @@ export const deployChange = async ({
   elementsSource?: ReadOnlyElementsSource
 }): Promise<ResponseResult> => {
   const instance = getChangeData(change)
+  log.debug(`Deploying instance ${instance.elemID.getFullName()}`, { action: change.action })
```

Comment:
you should add a `%s` for the 2nd parameter, otherwise it won't show up in the log. we should also probably make it consistent with the other parameters - e.g.
```suggestion
  log.debug('Deploying %s change for instance %s', change.action, instance.elemID.getFullName())
```
or
```suggestion
  log.debug(`Deploying ${change.action} change for instance ${instance.elemID.getFullName()}`
```

Reply:
It does show in the log, and i prefer to keep the start of the log 'Deploying <> instance' for easier searching